src/main/resources/logback.xml

```diff
@@ -9,15 +9,7 @@
         <thread>[ignore]</thread>
         <levelValue>[ignore]</levelValue>
       </fieldNames>
-      <providers>
-        <globalCustomFields>
-          <customFields>{"service":"banner-api"}</customFields>
-        </globalCustomFields>
-        <logLevel>
-          <fieldName>level</fieldName>
-          <fieldName>severity</fieldName>
-        </logLevel>
-      </providers>
+      <customFields>{"service":"banner-api"}</customFields>
```

Comment:
The config for `customField` has changed in this version of `LogstashEncoder` and no longer needs to be nested in a `providers.globalCustomFields`